public/app/plugins/datasource/prometheus/types.ts

```diff
@@ -32,6 +32,8 @@ export interface PromOptions extends DataSourceJsonData {
   prometheusType?: PromApplication;
   prometheusVersion?: string;
   defaultEditor?: QueryEditorMode;
+  incrementalQuerying?: boolean;
+  incrementalQueryOverlapWindow?: string; // ms
```

Comment:
I don't like to comment line but making that information explicit in the naming is not likable either.

Reply:
this comment is no longer true, actually. so it can be removed. we now parse these time strings, so it can be whatever.

Reply:
Yeah this should be removed, and has been!